ch09.adoc

```diff
@@ -488,7 +488,7 @@ Every feature within a Discrete Geometry CF file must be unambiguously associate
 
 
 
-Where feasible a variable with the attribute **cf_role** should be included.  The only acceptable values of cf_role for Discrete Geometry CF data sets are **`timeseries_id`** , **`profile_id`** , and **`trajectory_id`** .   The variable carrying the cf_role attribute may have any data type.  When a variable is assigned this attribute, it must provide a unique identifier for each feature instance.   CF files that contain timeSeries, profile or trajectory featureTypes, should include only a single occurrence of a **`cf_role`** attribute;  CF files that contain timeSeriesProfile or trajectoryProfile may contain two occurrences, corresponding to the two levels of structure in these feature types.
+Where feasible a coordinate or auxiliary coordinate variable with the attribute **cf_role** should be included.  The only acceptable values of cf_role for Discrete Geometry CF data sets are **`timeseries_id`** , **`profile_id`** , and **`trajectory_id`** .   The variable carrying the cf_role attribute may have any data type.  When a variable is assigned this attribute, it must provide a unique identifier for each feature instance.   CF files that contain timeSeries, profile or trajectory featureTypes, should include only a single occurrence of a **`cf_role`** attribute;  CF files that contain timeSeriesProfile or trajectoryProfile may contain two occurrences, corresponding to the two levels of structure in these feature types.
```

Comment:
Minor formatting issue: Use "\*\*cf_role\*\*" or "\*\* \`cf_role\` \*\*" instead of just "cf_role". This paragraph contains all three forms.